code/glTFAssetWriter.inl

```diff
@@ -105,16 +105,16 @@ namespace glTF {
                 type = "arraybuffer";
         }
 
-        obj.AddMember("byteLength", b.byteLength, w.mAl);
+        obj.AddMember("byteLength", (uint64_t)b.byteLength, w.mAl);
```

Comment:
Would a static_ast work as well. From my point of view a little bit more explicit,